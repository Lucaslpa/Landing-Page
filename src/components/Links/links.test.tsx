import { render } from 'utils/test-utils';
import { screen, fireEvent } from '@testing-library/react';
import { Links } from './index';

describe('Links', () => {
  it('Should render title', () => {
    render(<Links title="pastel" links={['link 1', 'link 2']} />);
    const title = screen.getByText(/pastel/gi);
    expect(title).toBeInTheDocument();
    expect(title).toMatchSnapshot();
  });

  it('Should render all links', () => {
    render(<Links title="pastel" links={['link 1', 'link 2']} />);
    const links = screen.getAllByRole('link');
    expect(links.length).toBe(2);
    expect(links).toMatchSnapshot();
  });
});
