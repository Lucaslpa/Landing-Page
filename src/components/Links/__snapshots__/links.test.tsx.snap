// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Links Should render all links 1`] = `
Array [
  @media (min-width:768px) {

}

@media (min-width:768px) {

}

<a
    href="/"
  >
    link 1
  </a>,
  @media (min-width:768px) {

}

@media (min-width:768px) {

}

<a
    href="/"
  >
    link 2
  </a>,
]
`;

exports[`Links Should render title 1`] = `
@media (min-width:768px) {

}

@media (min-width:768px) {

}

<span>
  pastel
</span>
`;
